test/extensions/KHR/SPV_KHR_cooperative_matrix/access_store_untyped.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_KHR_cooperative_matrix,+SPV_KHR_untyped_pointers -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv %t.spv -to-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: TypeInt [[#TypeInt:]] 32 0
+; CHECK-SPIRV-DAG: Constant [[#TypeInt]] [[#Const0:]] 0
+; CHECK-SPIRV-DAG: Constant [[#TypeInt]] [[#Const1:]] 1 {{$}}
+; CHECK-SPIRV-DAG: Constant [[#TypeInt]] [[#Const3:]] 3
+; CHECK-SPIRV-DAG: Constant [[#TypeInt]] [[#Const12:]] 12
+; CHECK-SPIRV-DAG: Constant [[#TypeInt]] [[#Const42:]] 42
+
+; CHECK-SPIRV: TypeCooperativeMatrixKHR [[#TypeMatrix:]] [[#TypeInt]] [[#Const3]] [[#Const12]] [[#Const12]] [[#Const0]]
+; CHECK-SPIRV: TypePointer [[#TypeMatrixPtr:]] 7 [[#TypeMatrix]]
+; CHECK-SPIRV: TypeUntypedPointerKHR [[#TypePtr:]] 7
+
+; CHECK-SPIRV: Variable [[#TypeMatrixPtr]] [[#VarMatrixPtr:]] 7
+; CHECK-SPIRV: CompositeConstruct [[#TypeMatrix]] [[#Composite:]] [[#Const0]]
+; CHECK-SPIRV: Store [[#VarMatrixPtr]] [[#Composite]]
+; CHECK-SPIRV: UntypedAccessChainKHR [[#TypePtr]] [[#Res:]] [[#TypeMatrix]] [[#VarMatrixPtr]] [[#Const1]]
+; CHECK-SPIRV: Store [[#Res]] [[#Const42]]
+
+; CHECK-LLVM: %0 = alloca target("spirv.CooperativeMatrixKHR", i32, 3, 12, 12, 0)
```

Comment:
```suggestion
; CHECK-LLVM: %[[#Matrix:]] = alloca target("spirv.CooperativeMatrixKHR", i32, 3, 12, 12, 0)
```
and so on

Reply:
It's basically just a copy of `access_store.ll` test :) do we need these checks there too?

Reply:
May be in another PR, not here :)